data/strings/types_strings.txt

```diff
@@ -9193,6 +9193,318 @@
     zh-Hans = 物理治疗师
     zh-Hant = 物理治療師
 
+  [type.healthcare.alternative]
+    en = Alternative medicine
+    ar = الطب البديل
+    be = Альтэрнатыўная медыцына
+    bg = Алтернативна медицина
+    cs = Alternativní medicína
+    da = Alternativ medicin
+    de = Alternative Medizin
+    el = Εναλλακτική ιατρική
+    es = Medicina alternativa
+    et = Alternatiivne meditsiin
+    eu = Medikuntza alternatiboa
+    fa = طب جایگزین
+    fi = Vaihtoehtoinen lääketiede
+    fr = Médecine alternative
+    he = רפואה אלטרנטיבית
+    hu = Alternatív gyógyászat
+    id = Pengobatan alternatif
+    it = Medicina alternativa
+    ja = 代替医療
+    ko = 대체 약품
+    mr = पर्यायी औषध
+    nb = Alternativ medisin
+    nl = Alternatieve geneeskunde
+    pl = Medycyna alternatywna
+    pt = Medicina alternativa
+    pt-BR = Medicina alternativa
+    ro = Medicină alternativă
+    ru = Альтернативная медицина
+    sk = Alternatívna medicína
+    sv = Alternativ medicin
+    sw = Dawa mbadala
+    th = การแพทย์ทางเลือก
+    tr = Alternatif tıp
+    uk = Альтернативна медицина
+    vi = Liều thuốc thay thế
+    zh-Hans = 替代药物
+    zh-Hant = 替代藥物
+
+  [type.healthcare.audiologist]
+    en = Audiology
+    ar = السمع
+    be = Аўдыялогія
+    bg = Аудиология
+    cs = Audiologie
+    da = Audiologi
+    de = Audiologie
+    el = Ακοολογία
+    es = Audiología
+    et = Audioloogia
+    eu = Audiologia
+    fa = شنوایی شناسی
+    fi = Audiologia
+    fr = Audiologie
+    he = אודיולוגיה
+    hu = Audiológia
+    id = Audiologi
+    it = Audiologia
+    ja = オーディオロジー
+    ko = 청력학
+    mr = ऑडिओलॉजी
+    nb = Audiologi
+    nl = Audiologie
+    pl = Audiologia
+    pt = Audiologia
+    pt-BR = Audiologia
```

Comment:
pt-BR = Fonoaudiologia

Reply:
Thanks changed in https://github.com/organicmaps/organicmaps/pull/4053/commits/dad75fe91b511fd26108fab402f7fc0a38781fe2